lotus/jsonrpc/client.go

```diff
@@ -85,12 +85,21 @@ type client struct {
 
```

Comment:
`lotus/jsonrpc` package is from Lotus. Has the changes proposed for autoconnect feature.
Can be dismissed for review... but if like crazy things might be fun.